Toolkit/Public/Get-RscNasSystem.ps1

```diff
@@ -23,33 +26,57 @@ function Get-RscNasSystem {
 
     .EXAMPLE
     Get details of NAS system with id="72859a28-6276-555a-9a66-d93fe99d2751"
+    Get-RscNasSystem -Id "72859a28-6276-555a-9a66-d93fe99d2751"
+        Or 
     Get-RscNasSystem "72859a28-6276-555a-9a66-d93fe99d2751"
 
     .EXAMPLE
-    Retrieve list of NAS systems.
+    Retrieve list of NAS systems (first 2).
```

Comment:
We don't need this. See comments below.

Reply:
Removed explicit mentioning of "first 2" here.